execution_engine/src/storage/store/mod.rs

```diff
@@ -58,9 +58,20 @@ pub trait Store<K, V> {
         K: AsRef<[u8]>,
         V: ToBytes,
         Self::Error: From<T::Error>,
+    {
+        self.put_raw(txn, key, &value.to_bytes()?)
+    }
+
+    /// Puts a raw `value` into the store at `key` within a transaction, potentially returning an
+    /// error of type `Self::Error` if that fails.
+    fn put_raw<T>(&self, txn: &mut T, key: &K, trie_bytes: &[u8]) -> Result<(), Self::Error>
```

Comment:
```suggestion
    fn put_raw<T>(&self, txn: &mut T, key: &K, bytes: &[u8]) -> Result<(), Self::Error>
```

Because it's not necessarily a trie, correct?

Reply:
No, it will always be a trie's bytes.

Reply:
It is part of `Store` interface which is implemented for all kinds of things. Just b/c it's called in the trie contexts right now does not mean we should be naming (otherwise generic) arguments with callers' contexts.

Reply:
> It is part of Store interface which is implemented for all kinds of things.

This function is specialized to the trie store, which only stores trie bytes, so the name seems appropriate.